pkg/dockermanager/dockermanager.go

```diff
@@ -49,6 +50,15 @@ func (dc *Container) Inspect(ctx context.Context) (types.ContainerJSON, error) {
 	return dc.Manager.ContainerInspect(ctx, dc.ID)
 }
 
+// Logs returns the logs for this docker container.
+func (dc *Container) Logs(ctx context.Context) (io.ReadCloser, error) {
+	return dc.Manager.ContainerLogs(ctx, dc.ID, types.ContainerLogsOptions{
```

Comment:
Why not replace `os.Stdout` and `os.Stderr` with teed I/O, such that it would duplicate the output to both the real stdout and stderr handles + `stdout.json` and `stderr.json` files, under the designated assets directory? The problem with this approach is that it won't work for the `local:exec` runner, no?